test/acceptance/features/addRBACForOperator.feature

```diff
@@ -0,0 +1,6 @@
+Feature: Making Service Biniding Operator RBAC pluggable so that other controllers/admins can add additional rules for the service binding operator.
```

Comment:
can we name `.feature` file to reflect this - e.g. `pluggableRBAC.feature`?

Reply:
yeah makes sense